### PR TITLE
make compattible with Cassandra 3.0

### PR DESCRIPTION
**Pull request by roongr2k7**
- remove varchar
- default time is localtime

### CODE REVIEW

**Comment by eladamitpxi** [comment]
please separate the hash commits from the cassandra 3 commits, i'd like to merge these to the cassandra 3 branch bump the plugin versions independently for each case

File: spec/unit/outputs/event_parser_spec.rb [hunk 1]
```diff
@@ -261,7 +260,7 @@
 
   describe 'ignore_bad_values is turned on' do
     [
-        { :name => 'timestamp', :value => 'i dont have to_time',      :expected => Time::parse('1970-01-01 00:00:00 +0000') },
```
Comment (eladamitpxi) [inline spec/unit/outputs/event_parser_spec.rb:263]: did this test fail for you?

Comment (roongr2k7) [reply spec/unit/outputs/event_parser_spec.rb:263]: @eladamitpxi yes

```
expected: "1970-01-01 00:00:00 UTC"
     got: "1970-01-01 07:00:00 +0700"
```

Comment (eladamitpxi) [reply spec/unit/outputs/event_parser_spec.rb:263]: strange that this didnt happen on my end, i'm on +0300 :)

**Comment by eladamitpxi** [comment]
looks good
just one last note, please target the dev-cass-3 branch in the PR

**Review by eladamitpxi — APPROVED** [review]

**Pull request merged**